index.html

```diff
@@ -245,35 +269,56 @@ <h1>About Me</h1>
-                    <div><a href="https://twitter.com/richardfriday14"><img src="./images/twitter.png" alt="twitter"></a></div>
-                    <div><a href="https://www.linkedin.com/in/richard-friday-54980718a"><img src="./images/Linkedin.png" alt="linkedin"></a></div>
-                    <div><a><img src="./images/medium.png" alt="medium"></a></div>
-                    <div><a href="https://github.com/Richez003"><img src="./images/github.png" alt="github"></a></div>
-                    <div><a href="https://angel.co/jobs?ref=onboarding"><img src="./images/angellist.png" alt="angellist"></a></div>
-                    </div>
-                    <a class="last-image" href="#"><img src="./images/illustration-contact-form.png" alt="illustrator"></a>     
-            </section>
-            <div>
-            <p class="lp">Richemonday@gmail.com</p>
-            </div>              
+            <div class="container-flex">
+                <div><a href="https://twitter.com/richardfriday14"><img src="./images/twitter.png" alt="twitter"></a></div>
+                <div><a href="https://www.linkedin.com/in/richard-friday-54980718a"><img src="./images/Linkedin.png" alt="linkedin"></a></div>
+                <div><a><img src="./images/medium.png" alt="medium"></a></div>
+                <div><a href="https://github.com/Richez003"><img src="./images/github.png" alt="github"></a></div>
+                <div><a href="https://angel.co/jobs?ref=onboarding"><img src="./images/angellist.png" alt="angellist"></a></div>
+          </div>
+                <a class="last-image" href="#"><img src="./images/illustration-contact-form.png" alt="illustrator"></a>     
+        </section>
+        <div>
+        <p class="lp">Richemonday@gmail.com</p>
+        </div>              
+            <div>  
+                    <a class="image5" href="#"><img src="./images/illustration-contact-form.png" alt="illustrator"></a>    
+            </div>
+          
+            <div class="illustrator">
+                <a href=""><img src="./images/illustration-1-contact-form-desktop.png" alt=""></a>
+                <a href="" class="frame3"><img src="./images/Frame3.png" alt="#"></a>
+                </div> 
+    
+                <div class="flex-container1">
+                        <div><a href="https://twitter.com/richardfriday14"><img src="./images/twitter.png" alt="twitter"></a></div>
+                        <div><a href="https://www.linkedin.com/in/richard-friday-54980718a"><img src="./images/Linkedin.png" alt="linkedin"></a></div>
+                        <div><a><img src="./images/medium.png" alt="medium"></a></div>
+                        <div><a href="https://github.com/Richez003"><img src="./images/github.png" alt="github"></a></div>
+                        <div><a href="https://angel.co/jobs?ref=onboarding"><img src="./images/angellist.png" alt="angellist"></a></div>
+            </div>             
+           
```

Comment:
- [OPTIONAL ] Kindly add this vertical line before your social icons

- Your Design

![ba2](https://user-images.githubusercontent.com/79058364/161384033-38aadf9a-c581-42d4-8175-c3b61595fd76.png)

- Figma design

![go5555](https://user-images.githubusercontent.com/79058364/161379500-7e3c335c-ade7-48aa-9e19-270cfb6dfbab.png)